packages/smb-server/src/server/broker.class.ts

```diff
@@ -0,0 +1,50 @@
+import { randomUUID } from 'crypto';
+import { createServer, Server } from 'net';
+import { Injectable } from 'peque-di';
+import { BrokerSocket, Command, EventService, SocketService } from 'peque-smb-commons/src';
+
+import { MessageCommand, PublishCommand, SubscribeCommand, UnsubscribeCommand, WelcomeCommand } from '../commands';
+
+@Injectable()
+export class Broker {
+  #server: Server;
+  #port = Number(process.env.PORT) ?? 8888;
+  #hostname: string = process.env.HOSTNAME ?? '127.0.0.1';
```

Comment:
For env variables it is safer to use `||` instead of `??`. That would cover for when the value is either an empty string or 0 (like in `Number(process.env.PORT)`)
```suggestion
  #port = Number(process.env.PORT) || 8888;
  #hostname: string = process.env.HOSTNAME || '127.0.0.1';
```